README.md

```diff
@@ -2,10 +2,10 @@
 
 The adapter was initially developed for conjugate heat transfer (CHT) simulations via preCICE by Lucia Cheung in the scope of her master’s thesis [[1]](https://www5.in.tum.de/pub/Cheung2016_Thesis.pdf) in cooperation with [SimScale](https://www.simscale.com/). For running the adapter for CHT simulations refer to this thesis. The adapter was extended to fluid-structure interaction by Alexander Rusch [[2]](https://www.gacm2017.uni-stuttgart.de/registration/Upload/ExtendedAbstracts/ExtendedAbstract_0138.pdf).
 
-This adapter was developed for **CalculiX version 2.16.** Other versions may be compatible, yet they have not been tested. Please let us know if you want to use a different version.
+This adapter was developed for **CalculiX version 2.17.** Other versions may be compatible, yet they have not been tested. Please let us know if you want to use a different version.
 
 Adapters for other versions of CalculiX and preCICE are available in various branches. Branches compatible with **preCICE v2.x:**
- - master
+ - master (currently still for CalculiX v2.16)
  - v2.15_preCICE2.x
```

Comment:
Yes. But I'd change the overall README as "This (master) branch currently supports 2.19. Other branches are legacy, and preCICE2 is required for >= 2.15" instead of a list. What de you think ?

Reply:
I guess we can even remove any mention to non-preCICE2 anyway. People that are still using preCICE1 are not going to upgrade any adapter anyway.